src/customEditors/EditorSensorItem.tsx

```diff
@@ -65,20 +68,58 @@ export const EditorSensorItem: React.FC<Props> = (props: Props) => {
 
       <Field label="Name">
         <Input
+          css=""
+          key={sensor.name}
           value={sensor.name}
-          onChange={event => {
-            updateSensor(sensor => {
+          onChange={(event) => {
+            updateSensor((sensor) => {
               sensor.name = event.currentTarget.value;
             });
           }}
         />
       </Field>
+      <Field label="Name display">
+        <Switch
+          css=""
+          value={sensor.nameDisplay}
+          onChange={(event) => {
+            updateSensor((sensor) => {
+              sensor.nameDisplay = event.currentTarget.checked;
+            });
+          }}
+        />
+      </Field>
+      <Field label="Value display">
+        <Switch
+          css=""
+          value={sensor.valueDisplay}
+          onChange={(event) => {
+            updateSensor((sensor) => {
+              sensor.valueDisplay = event.currentTarget.checked;
+            });
+          }}
+        />
+      </Field>
+
+      <Field label="IconName">
```

Comment:
```suggestion
      <Field label="Icon" description="Write a valid FontAwesome icon">
```

I noticed that some icons like `amazon` from here https://fontawesome.com/icons don't work, do you know why?
Do you think it's possible to retrieve a list of possible icons from the library and present it as a select?